desktop-app/app/components/WebView/index.js

```diff
@@ -185,6 +185,13 @@ class WebView extends Component {
     );
 
     this.webviewRef.current.addEventListener('dom-ready', () => {
+      this.getWebContentForId(this.webviewRef.current.getWebContentsId())
+        .executeJavaScript(
+          `{
+            window._bot = true;
```

Comment:
This runs after `dom-ready`, what if the analytics script executed before this? 🤔

Reply:
If I move it outside `dom-ready` event (i.e.: the start of `componentDidMount`) I get:
![image](https://user-images.githubusercontent.com/13673443/176872287-e9ee5a10-7405-4ee1-8b50-832b7808f090.png)

So, I moved the `window._bot = true` line to `preload.js` and it seems to work

what do you think @manojVivek?

Reply:
Yes, this looks good JJ!